pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -686,6 +676,18 @@ func allocatableResources(memory, cpu string) v1.ResourceList {
 
 // DaemonSets should not place onto nodes with insufficient free resource
 func TestInsufficientCapacityNodeDaemonDoesNotLaunchPod(t *testing.T) {
+	// We don't need this test as default scheduler is responsible for doing resource checks when ScheduleDaemonSetPods
+	// is enabled. We are unsetting ScheduleDaemonSetPods to test codepath without ScheduleDaemonSetPods.
```

Comment:
Please put the test description before the test, similar to what we had for `TestSimpleDaemonSetScheduleDaemonSetPodsLaunchesPods`, and rephrase it: 

```
// When ScheduleDaemonSetPods is disabled, DaemonSets should not place onto nodes with insufficient free resource
```

This comment applies to other tests as well.